lib/App/perlimports/Document.pm

```diff
@@ -400,12 +401,40 @@ sub _build_possible_imports {
 
         next if !$isa_symbol && is_method_call($word);
 
-        # A hash key might, for example, be a variable.
+        # The => operator (sometimes pronounced "fat comma") is a synonym for
+        # the comma except that it causes a word on its left to be interpreted
+        # as a string if it begins with a letter or underscore and is composed
+        # only of letters, digits and underscores. This includes operands that
+        # might otherwise be interpreted as operators, constants, single number
+        # v-strings or function calls.
+        # https://perldoc.perl.org/perlop#Comma-Operator
+        my $word_is_string;
+        if (   $word->statement->isa('PPI::Statement::Expression')
+            && $word->isa('PPI::Token::Word')
+            && $word->content =~ /^[a-zA-Z_][a-zA-Z0-9_]*$/ ) {
+            my @children = $word->statement->children;
```

Comment:
If we use `$word->statement->schildren;` here rather than `$word->statement->children;`, I think we don't need the `next if $current->isa('PPI::Token::Whitespace');` below.